source/PyMaterialX/PyGenOptions.cpp

```diff
@@ -0,0 +1,23 @@
+//
+// TM & (c) 2019 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+#include <PyMaterialX/PyMaterialX.h>
+
+#include <MaterialXGenShader/GenOptions.h>
+
+namespace py = pybind11;
+namespace mx = MaterialX;
+
+void bindPyGenOptions(py::module& mod)
+{
+    py::enum_<mx::ShaderInterfaceType>(mod, "ShaderInterfaceType")
+        .value("SHADER_INTERFACE_COMPLETE", mx::ShaderInterfaceType::SHADER_INTERFACE_COMPLETE)
+        .value("SHADER_INTERFACE_REDUCED", mx::ShaderInterfaceType::SHADER_INTERFACE_REDUCED);
+
+    py::class_<mx::GenOptions>(mod, "GenOptions")
+        .def_readwrite("shaderInterfaceType", &mx::GenOptions::shaderInterfaceType)
+        .def_readwrite("hwTransparency", &mx::GenOptions::hwTransparency)
```

Comment:
I'm wondering if we should fix this name in C++ and here. hw is kind-of meaningless unless your familiar with it. Maybe use the full string "hardware" ?

Reply:
That's fine by me, I'll make the change.

Reply:
Actually as discussed, we use hw in a bunch of places, so leaving this alone for now.